### ISSUE TITLE
some CHEBI classes do not have any parent class

### ISSUE THREAD

**Issue by laurent2207** [enanomapper/ontologies#229]
I noted that the following CHEBI classes do not have any parent class and do not show up on bioportal unless you specifically search for them.

It is about the following CHEBI classes:
CHEBI 27569: selenium atom
CHEBI 27638: cobalt atom
CHEBI 28685: molybdenum atom
CHEBI 31794: magnesium oxide
CHEBI 33341: titanium atom
CHEBI 51110: copper(II) sulfide
CHEBI 60533: γ-poly(L-glutamic acid) polymer
CHEBI 85541: elemental molecular entity

structure as seen in Protegé:
![image](https://user-images.githubusercontent.com/44968488/81565474-ae31cc80-9399-11ea-9af9-a2e3462a4b15.png)

Example of specific search for one of these classes:
![image](https://user-images.githubusercontent.com/44968488/81565516-c0ac0600-9399-11ea-90e9-dbd463f55ea8.png)

I do not know if we should manually attend parent classes to these, or how to tackle this problem!?

**Comment by egonw** [comment]
I have pushed a patch that replaces the 'atom' parts with their respective 'molecular entity' parts. Let's see how that goes,

**Comment by egonw** [comment]
So, nanoparticles have a chemical content, they consist of material. So, first they consisted of atoms, not bad, but those atoms were always neutral (well, 0 valency) and in retrospect maybe that may not be the best representation. So, I first updated the affected classes to have an axiom mentioning the corresponding materials. In the second step I added these materials to our ontology, under the right super class.
